src/components/Forms/Annocement/annoucement.cropper.tsx

```diff
@@ -0,0 +1,71 @@
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import { useCallback, useEffect, useState } from 'react';
+import Cropper from 'react-easy-crop';
+import { Area } from 'react-easy-crop/types';
+import DialogTitle from '../../DialogTitle';
+import { getCroppedImg } from './annoucement.util';
+import { StyledButton, StyledDialogContent } from './styles';
+
+const CropperDialog = ({
+  open,
+  closeModal,
+  currentImage,
+  images,
+  setFieldValue,
+}: {
+  open: boolean;
+  closeModal: () => void;
+  currentImage: string | null;
+  images: string[];
+  setFieldValue: (field: string, value: any, shouldValidate?: boolean | undefined) => void;
+}) => {
```

Comment:
It's a quite big object of types, I suggest creating the external interface for it.